tests/Endpoints/DocumentsTest.php

```diff
@@ -280,6 +302,34 @@ public function testAddDocumentsCsvInBatches(): void
         $this->assertSame($total, $response->getTotal());
     }
 
+    public function testAddDocumentsCsvInBatchesWithDelimiter(): void
```

Comment:
would it fail with empty delimiter? if yes, maybe better to document then as `non-empty-string|null` ?

Reply:
With this phpdoc users could know what it would fail even before calling the api

Reply:
Yes, thanks @alallema I'll also add your suggestion @norkunas :)